var/spack/repos/builtin/packages/orca/package.py

```diff
@@ -49,13 +49,15 @@ class Orca(Package):
     )
 
     depends_on("zstd", when="@:4.2.1", type="build")
+    depends_on("libevent", type="run")
+    depends_on("libpciaccess", type="run")
 
     # Map Orca version with the required OpenMPI version
     openmpi_versions = {
         "4.0.1.2": "2.0.2",
         "4.2.0": "3.1.4",
         "4.2.1": "3.1.4",
-        "5.0.3": "4.1.2",
+        "5.0.3": "4.1.1",
```

Comment:
4.1.2 was specified previously due ot a build error I was seeing with 4.1.1 specifically [in the past](https://github.com/spack/spack/pull/33649#issue-1432011610)

Reply:
We had some memory issues running with the previous build in parallel. After adding the libraries to the runtime environment and changing the `openmpi` version, the problems were gone. After adding the libraries to the runtime environment, I must admit that I still need to check 4.1.2. However,, guided by my past experience with `orca,` which proved to be very stubborn to the minor version of `openmpi,` I proposed the change. If it should work for any 4.1.x, should we aim for this spec?

Reply:
I've confirmed that the build error still exists with openmpi@4.1.1 in my environment. If you can test that 4.1.2 works for you I'd appreciate it. I'm not opposed to switching it to 4.1.1 and dealing with it on our end, since I suspect the issue only exists in pmix environments, which probably isn't everyone.

Reply:
Let's do the following. I will remove the change of version from this PR and check 4.1.2. I will come back to the issue in a different PR if the problem persists.

Reply:
sounds good to me.